ui/src/containers/AlertHistoryProvider.js

```diff
@@ -0,0 +1,32 @@
+import React, { createContext, useContext } from 'react';
+import { useQuery } from 'react-query';
+import { getLast7DaysAlerts } from '../services/loki/api';
+import { filterAlerts } from '../services/alertUtils';
+
+const AlertHistoryContext = createContext(null);
+
+export function useHistoryAlerts(filters?: FilterLabels) {
+  const query = useContext(AlertHistoryContext);
+  if (query.status === 'success') {
```

Comment:
I think a null check on query is missing here and should throw an exception to indicate that this hook can only be used within an AlertHistoryProvider